### DISCUSSION TITLE
Re-new service : aws-lambda-runner

### DISCUSSION THREAD

**Discussion by lacell75** [webdriverio/webdriverio#9644] [Ideas]
I think to resume the development of aws-lambda-runner would be much appreciated . 

The benefit of serverless mode is very important, especially when there is a large volume of tests.
Currently, we are encountering many maintenance or ressources problems on our server infrastructure (grid + nodes) because the number of machines became huge which causes a lot of instability.
I think the lambda can solve this problematic.

**Comment by christian-bromann** [comment]
> I think the lambda can solve this problematic.

I agree, a Lambda runner could be a cool grid alternative. Feel free to take a stab at it. We are happy to support you in any way needed, e.g. tech support or testing.
